docs/learn/tokenomics2/tokenomics2-network-fees.md

```diff
@@ -134,4 +134,24 @@ The values for the parameters above are set so that EVM fee and the Native fee a
 
 ## Fee Alignment Transition Period
 
-Legacy Astar Network tokenomics fee model was not aligned between the two systems - same resource consumption via native or Ethereum transactions resulted in significantly different fees. To allow network stakeholders to adjust to the Tokenomics 2.0 fee model, alignment of fees between the two systems will be gradually introduced once the change is enacted (live) on the network.
\ No newline at end of file
+Legacy Astar Network tokenomics fee model was not aligned between the two systems - same resource consumption via native or Ethereum transactions resulted in significantly different fees. To allow network stakeholders to adjust to the Tokenomics 2.0 fee model, alignment of fees between the two systems will be gradually introduced once the change is enacted (live) on the network.
+
+## Hybdrid Inflation model
+
+The new inflation model will be preceded by an intermediate phase known as the hybrid inflation model. This transitional phase encompasses these modifications:
+
+### Inflation Adjustment
+
+Inflation rates have been lowered. The maximum token reward per block, reached based on the optimal staking rate or staking TVL (Total Value Locked), has been reduced from 253.08 to 230.71.
```

Comment:
Could you please add some text about this being a short temporary measure since we'll transition over to Tokenomics 2.0 very soon.